micros/setting/services/userSettingService.js

```diff
@@ -7,13 +7,23 @@ const { v4: uuidv4 } = require("uuid");
 // const MUUID = require("uuid-mongodb");
 
 exports.saveManyUserSetting = async function (userSettingList) {
+  let bulkItem = [];
   for (const setting of userSettingList) {
     if (setting.objectId == "") setting.objectId = uuidv4();
 
     if (setting.created_date == "")
       setting.created_date = Math.floor(Date.now() / 1000);
-    return UserSetting(setting).save();
+    bulkItem.push(setting);
   }
+  UserSetting.insertMany(bulkItem)
```

Comment:
Dear @akcpu

Thank you for nice implementation👍
To unified the code and preventing unexpected errors, please use `async/await - try/catch` only instead of `Promise.then().catch()` unless there is a special case you have to handle it by `Promise.then().catch()`.